src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
 'use strict';
 
-// write your code here
+const allPeople = document.querySelectorAll('.population');
+
+let total = 0;
+
+for (const eachCountry of allPeople) {
+  total += +eachCountry.innerText.split(',').join('');
+}
+
+const average = total / allPeople.length;
+
+const totalResult = [];
+const averageResult = [];
+let subCounterTotal = 0;
+let subCounterAverage = 0;
+
+for (let i = String(total).length - 1; i >= 0; i--) {
+  subCounterTotal++;
+
+  if (subCounterTotal === 4) {
+    totalResult.unshift(',');
+    i++;
+    subCounterTotal = 0;
+  } else {
+    totalResult.unshift(String(total)[i]);
+  }
+}
+
+for (let i = String(average).length - 1; i >= 0; i--) {
+  subCounterAverage++;
+
+  if (subCounterAverage === 4) {
+    averageResult.unshift(',');
+    i++;
+    subCounterAverage = 0;
+  } else {
+    averageResult.unshift(String(average)[i]);
+  }
+}
+
+document.querySelector('.total-population')
+  .innerText = totalResult.join('');
+
+document.querySelector('.average-population')
+  .innerText = averageResult.join('');
```

Comment:
```suggestion
document.querySelector('.total-population')
  .innerText = total.toLocaleString('en-US');

document.querySelector('.average-population')
  .innerText = average.toLocaleString('en-US');
```

Clever solution. But you could use `.toLocaleString` method instead